File: src/NeuralNetwork.py
import random
from const import *

class Neuron:
    def __init__(self,neuron_id):
        #dynamic
        activation = 0.0
        #genetic
        activation_bias = random.choice([-1,0,1])
        coefficient = random.choice([1,4,9])

class Connection:
    def __init__(self):
        #dynamic
        weight = 0.0
        #genetic
        innate_weight = random.uniform(-1.0,1.0)
        input_neuron_index = random.randint(-1, H + O + i)
        neuromodulation_neuron_index
        neurotransmission_neuron_index
        output_neuron_index

class NeuralNetwork:
    def __init__(self):
        

File: src_cpp/neural_network.cpp
#include "Random.cpp"
#include<iostream>

class neuron
{
    private:
    float activation_val = 0.0; // -1.0 ~ +1.0
    int activation_bias;
    int coefficient;

    public:
    neuron()
    {
        int flag = rnd(3);
        if(flag == 0){ activation_bias = -1; }
        else if (flag == 1){ activation_bias = 0; }
        else { activation_bias = 1; }

        flag = rnd(3);
        if(flag == 0){ coefficient = 1; }
        else if (flag == 1){ coefficient = 4; }
        else { coefficient = 9; }
    }
};

int main()
{
    neuron n[10];
    for(int i=0;i<10;i++)
    {
        std::cout << n[i].activation_val << std::endl;
        std::cout << n[i].activation_bias << std::endl;
    }

    return 0;
}
